decidim-comments/app/frontend/comments/comment.component.jsx

```diff
@@ -1,54 +1,172 @@
-import { propType }    from 'graphql-anywhere';
-import gql             from 'graphql-tag';
-import UserAvatar      from 'react-user-avatar';
-import moment          from 'moment';
+import { Component, PropTypes } from 'react';
+import { propType }             from 'graphql-anywhere';
+import gql                      from 'graphql-tag';
+import UserAvatar               from 'react-user-avatar';
+import moment                   from 'moment';
+import { I18n }                 from 'react-i18nify';
 
-import commentFragment from './comment.fragment.graphql'
+import AddCommentForm           from './add_comment_form.component';
+
+import commentFragment          from './comment.fragment.graphql';
+import commentDataFragment      from './comment_data.fragment.graphql';
 
 /**
- * @returns {ReactComponent} - A single comment component with the author info and the comment's body
+ * A single comment component with the author info and the comment's body
+ * @class
+ * @augments Component
  */
-const Comment = ({ comment: { author, body, createdAt } }) => {
-  let authorInitialLetter = " ";
-  const formattedCreatedAt = ` ${moment(createdAt, "YYYY-MM-DD HH:mm:ss z").format("LLL")}`;
-  
-  if (author.name.length > 0) {
-    authorInitialLetter = author.name[0];
+class Comment extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showReplyForm: false
+    };
   }
 
-  return (
-    <article className="comment">
-      <div className="comment__header">
-        <div className="author-data">
-          <div className="author-data__main">
-            <div className="author author--inline">
-              <span style={{ color: "#fff" }} className="author__avatar">
-                <UserAvatar size="20" name={authorInitialLetter} />
-              </span>
-              <a className="author__name">{author.name}</a>
-              <time dateTime={createdAt}>{formattedCreatedAt}</time>
+  render() {
+    const { comment: { id, author, body, createdAt }, articleClassName } = this.props;
+
+    let authorInitialLetter = " ";
+    const formattedCreatedAt = ` ${moment(createdAt, "YYYY-MM-DD HH:mm:ss z").format("LLL")}`;
+    
+    if (author.name.length > 0) {
+      authorInitialLetter = author.name[0];
+    }
+
+    return (
+      <article id={`comment_${id}`} className={articleClassName}>
+        <div className="comment__header">
+          <div className="author-data">
+            <div className="author-data__main">
+              <div className="author author--inline">
+                <span style={{ color: "#fff" }} className="author__avatar">
+                  <UserAvatar size="20" name={authorInitialLetter} />
```

Comment:
Use the avatars (you might need to expose the `current_user.avatar.url` through the API)